extensions/playerlist/in-div.ejs

```diff
@@ -4,6 +4,13 @@
   }
 </style>
 
+<script type="text/javascript">
+  function startPrivateMessage(playerID) {
+    console.log(playerID);
+    document.getElementById("message_to").value = playerID;
```

Comment:
Optional: Would recommend moving this to client.mjs to make it simpler when the apply the UI changes.